app/javascript/containers/RebateForm.js

```diff
@@ -80,16 +80,19 @@ class EditRebateForm extends React.Component {
         }) => {
           const {
             fields:
-            { spouse_or_partner,
+            { incomeLessThan5k,
```

Comment:
should this be snake_case like the others, instead of camelCase?

Reply:
its come through from Juan

Reply:
😞